### ISSUE TITLE
usage of sigmoid_focal_crossentropy in graph mode

### ISSUE THREAD

**Issue by travishsu** [tensorflow/addons#2116]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.03 in Docker 
- TensorFlow version and how it was installed (source or binary): 2.1
- TensorFlow-Addons version and how it was installed (source or binary): 0.9.0
- Python version: 3.6.9
- Is GPU used? (yes/no): yes

**Describe the bug**

I have `PartitionedCall` issue while calling `tfa.losses.sigmoid_focal_crossentropy` in Keras Function API mode.

**Code to reproduce the issue**

https://colab.research.google.com/drive/1hlfx9tLRk9VQ4oewED7joqOs8e6Sdtk4?usp=sharing

**Other info / logs**

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by bhack** [comment]
What are you trying to do? I see your passing the loss as the 2nd argument of `Model`?
Check `Model` arguments https://www.tensorflow.org/api_docs/python/tf/keras/Model

**Comment by travishsu** [comment]
Hi, I’m trying to adapt that to matterport/Mask_RCNN

https://github.com/matterport/Mask_RCNN/blob/master/mrcnn/model.py#L2026

**Comment by bhack** [comment]
What I meant is that generally the loss is passed in the `model.compile`. So I don't think that your example is a bug.

**Comment by travishsu** [comment]
Ok, I see. Thx.
